pkg/traceutil/trace.go

```diff
@@ -22,9 +22,19 @@ import (
 	"strings"
 	"time"
 
+	"go.opentelemetry.io/otel/trace"
+	"go.opentelemetry.io/otel/trace/noop"
 	"go.uber.org/zap"
 )
 
+const instrumentationScope = "go.etcd.io/etcd"
+
+var Tracer trace.Tracer = noop.NewTracerProvider().Tracer(instrumentationScope)
+
+func Init(tp trace.TracerProvider) {
+	Tracer = tp.Tracer(instrumentationScope)
```

Comment:
maybe we should introduce syncOnce for this tracer. just a case that it could be race-condition. it's not blocker

Reply:
I tried, but unfortunately it breaks tracing integration tests (`Tracer` needs to be overwritten for each test case)